client/src/components/css/QA.css

```diff
@@ -12,42 +12,49 @@ td {
   vertical-align: top;
 }
 
-.character {
+.QA_character {
   font-weight: bold;
 }
-.answer_by {
+.QA_answer_by {
   margin: 5px;
   font-size: 14px;
 }
 
-.add_question, .more_questions {
+.QA_add_question, .QA_more_questions {
   height: 50px;
   margin: 8px;
   font-weight: 550 ; 
   display: inline-block; 
 }
 
-.add_question {
+.QA_add_question {
   width: 200px
 }
-.more_question {
+.QA_more_question {
   width: 250px
 }
 
-.question_body{
+.QA_question_body{
   font-weight: bold;
+  font-size: 20px;
```

Comment:
Try not to use absolute values for CSS. It's better to use relative measurements such as 'em' (for font)

Reply:
I'm confused when to use em or px, I will edit CSS styling and make new PR